// This should produce an error: break outside loop
print "This will error";
break;